File: Arrays/Triangle/code.js
// you can write to stdout for debugging purposes, e.g.
// console.log('this is a debug message');

function solution(A) {
    // write your code in JavaScript (Node.js 8.9.4)
    A.sort((a, b) => a - b);
    //console.log(A);
    if (A.length < 3)
        return 0;
    for (var i = 2; i < A.length; i++) {
        if (((A[i - 1] + A[i - 2]) > A[i]) && ((A[i - 2] + A[i]) > A[i - 1]) && ((A[i - 1] + A[i]) > A[i - 2]))
            return 1;
    }
    return 0;

}

File: Greedy/MaxNonoverlappingSegments/code.java
// you can also use imports, for example:
// import java.util.*;

// you can write to stdout for debugging purposes, e.g.
// System.out.println("this is a debug message");

class Solution {
    public int solution(int[] A, int[] B) {
        // write your code in Java SE 8
        int pre = -1, ans = 0;
        for (int i = 0; i < A.length; i++) {
            if ((pre < 0) || (A[i] > B[pre])) {
                pre = i;
                ans++;
            }

        }
        return ans;
    }
}